server/src/routes/api/v1/podcastsRouter.js

```diff
@@ -23,11 +24,14 @@ podcastsRouter.get("/:id", async (req, res) => {
   try {
     const { id } = req.params
     const podcast = await Podcast.query().findById(id)
+    podcast.reviews = await podcast.$relatedQuery('reviews')
```

Comment:
ideally this logic should be added to the serializer when we retrieve all of the properties we want